### PR TITLE
0.12.0 rc into master

### PR DESCRIPTION
**Pull request by MatthewSandfordImprobable**
Merging the RC back into master to stop it diverging too much.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -118,6 +122,8 @@ These functions and structs can be referenced in both code and blueprints it may
 - Fixed a rare crash where a RepNotify callback can modify a GDK data structure being iterated upon.
 - Fixed race condition in Spatial Test framework that would cause tests to time out with one or more workers not ready to begin the test.
 - Fixed client connection not being cleaned up when moving out of interest of a server.
+- Fixed a assertion being triggered on async loaded entities due to queuing some component addition.
```
Comment (mironec) [inline CHANGELOG.md:125]: Are these new changelog entries correct? New bugfixes in the rc being brought over into master?

Comment (MatthewSandfordImprobable) [reply CHANGELOG.md:125]: This is from Nics PR into RC: https://github.com/spatialos/UnrealGDK/pull/2770

**Review by ImprobableNic — APPROVED** [review]

File: ci/unreal-engine.version [hunk 2]
```diff
@@ -1,3 +1,3 @@
 HEAD 4.26-SpatialOSUnrealGDK
 HEAD 4.25-SpatialOSUnrealGDK
-HEAD 4.24-SpatialOSUnrealGDK
+HEAD 4.24-SpatialOSUnrealGDK
```
Comment (m-samiec) [inline ci/unreal-engine.version:3]: This change okay? I know our CI gets fussy about EOF lines etc. cc: @mironec

**Review by m-samiec — APPROVED** [review]

File: ci/unreal-engine.version (hunk 2 above)
Comment (mironec) [reply ci/unreal-engine.version:3]: Should be okay either way in this file, but I guess if it does not have to change, don't change it.

**Review by mironec — APPROVED** [review]

**Pull request merged**